### PR TITLE
feat(vendor-core): update pf4 to 2021.02 version

### PR DESCRIPTION
**Pull request by MariaAga**

### CODE REVIEW

**Comment by MariaAga** [comment]
@johnpmitsch @amirfefer Can you take a look/merge?

File: packages/vendor-core/package.json [hunk 1]
```diff
@@ -24,13 +24,13 @@
...
     "@apollo/client": "^3.3.7",
     "@novnc/novnc": "novnc/noVNC#25b3d49d322b0a7c9ee1e071d93042d70f5176b7 ",
-    "@patternfly/patternfly": "^4.70.2",
-    "@patternfly/react-catalog-view-extension": "^4.8.126",
-    "@patternfly/react-core": "^4.84.4",
-    "@patternfly/react-icons": "^4.7.22",
-    "@patternfly/react-styles": "^4.7.22",
-    "@patternfly/react-table": "^4.19.45",
-    "@patternfly/react-tokens": "^4.9.22",
+    "@patternfly/patternfly": "^4.65.5",
```
Comment (johnpmitsch) [inline packages/vendor-core/package.json:27]: @MariaAga should this be lower than the previous version?

Comment (MariaAga) [reply packages/vendor-core/package.json]: Nope, bad rebase, removed

**Review by johnpmitsch — APPROVED** [review]
Tests pass and nothing broke on a quick smoke test! :+1:

**Pull request merged**